go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/PuerkitoBio/goquery v1.9.2
 	github.com/avast/retry-go v3.0.0+incompatible
 	github.com/aws/aws-sdk-go v1.55.5
```

Comment:
What references are remaining that cause us to still be referencing `aws/aws-sdk-go`?